web-components-f902/notes/00_javascript-render-hello-world-text-with-custom-elements.md.md

```diff
@@ -0,0 +1,33 @@
+# Render Hello World Text with Custom Elements
+
+[Video link](https://egghead.io/lessons/javascript-render-hello-world-text-with-custom-elements)
+
+<TimeStamp start="0:09" end="0:17">
+
+We first define a class, `MyCustomElement()` which extends `HTMLElement` inside the `<script>` tags before creating our custom element.
+
+</TimeStamp>
+
+<TimeStamp start="0:29" end="0:37">
+
+We define a `connectedCallback` after defining our click event set up with a callback. A `connectedCallback` is invoked each time the custom element is appended into a document-connected element.
```

Comment:
I'd shorten this to just "A `connectedCallback` is invoked each time the custom element is appended into a document-connected element."